.github/workflows/vnu-jar-problem-matcher.json

```diff
@@ -4,12 +4,12 @@
       "owner": "vnu-jar",
       "pattern": [
         {
-          "regexp": "^\"file:(.*)\":(\\d+).(\\d+)-\\d+\\.\\d+:\\s(error|warning):\\s(.*)$",
+          "regexp": "^\"file:(.*)\":(\\d+).(\\d+)-\\d+\\.\\d+:\\s(info )?(error|warning):\\s(.*)$",
```

Comment:
Is it possible to use non-capturing `(?:info )` instead?

Reply:
No, because it only really recognized the Warning/Error levels for the metadata you see on the files tab

Reply:
I don't understand. Is the `?:` regexp syntax not supported?

Reply:
Sorry, I thought you meant capturing the "info" instead of the warning. Not sure about the updated capture group, but I could try later

Reply:
Looks like it works!